### PR TITLE
slowHand14 PR

### PR DESCRIPTION
**Pull request by paparoach14**

### CODE REVIEW

**Review by yeelali14 — COMMENTED** [review]
test

**Review by yeelali14 — CHANGES_REQUESTED** [review]
please

**Review by yeelali14 — CHANGES_REQUESTED** [review]
yep

**Review by yeelali14 — COMMENTED** [review]
check

**Review by slowHand14 — COMMENTED** [review]
jh

**Review by yeelali14 — COMMENTED** [review]
eee

File: README.md [hunk 1]
```diff
@@ -17,3 +17,4 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
 
 #PR
```
Comment (yeelali14) [inline README.md:19]: ok

Comment (yeelali14) [reply README.md:19]: test

**Review by yeelali14 — CHANGES_REQUESTED** [review]

**Review by yeelali14 — APPROVED** [review]

**Review by yeelali14 — COMMENTED** [review]
ttt

File: README.md [hunk 2]
```diff
@@ -17,3 +17,4 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
...
+test
```
Comment (yeelali14) [inline README.md:20]: why

**Pull request closed without merging**